.lint_glob_excludes

```diff
@@ -0,0 +1 @@
+python/build/**
```

Comment:
what happens if you don't include this?

Reply:
I think build is from pypi builds? So pylint will complain about it. but it's just copies of what's in src. same with `dist`. I'm actually not sure why there's both dist and build.